build/build_versions.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-set -e
+set -x
```

Comment:
Maybe you changed this just for debugging or do you wish to actually commit it ?